pandas/tests/series/methods/test_rename.py

```diff
@@ -101,3 +102,16 @@ def test_rename_callable(self):
         tm.assert_series_equal(result, expected)
 
         assert result.name == expected.name
+
+    def test_rename_method_and_index(self):
+        # GH 40977
+        s = Series([1, 2])
```

Comment:
nitpick: can we avoid 1-letter variable names?

Reply:
sure, thanks (might be worth adding such a rule to pandas' flake8 plugin)